showcase.json

```diff
@@ -5164,5 +5164,27 @@
     "twitter": "https://x.com/CoreLedger",
     "usesInternetIdentity": true,
     "display": "Normal"
+  },
+  {
+    "name": "BeProActive",
+    "description": "The dApp helps you to track your daily workouts and share them with the community and your friends.",
+    "website": "https://beproactive.at",
+    "logo": "/img/showcase/beproactive_logo.webp",
+    "screenshots": [
+      "/img/showcase/beproactive_screenshot.webp"
+    ],
+    "video": "",
+    "display": "Normal",
+    "id": "beproactive",
+    "oneLiner": "Track and share your daily workouts with the community and friends.",
+    "stats": "4+",
+    "tags": [
+      "SocialFi"
+    ],
+    "usesInternetIdentity": true,
+    "github": "https://github.com/samlinux-development/beProActive",
+    "twitter": "",
```

Comment:
Please remove any key-value pairs that are empty.

Twitter
Youtube
submittable ID
video